v3/deployment.go

```diff
@@ -76,19 +78,9 @@ var _ = V3Describe("deployment", func() {
 				g.Expect(session).Should(Exit(0))
 			}).Should(Succeed())
 
-			counter := 0
-			Eventually(func() int {
-				if strings.Contains(helpers.CurlAppRoot(Config, appName), "Hello from a staticfile") {
-					counter++
-				} else {
-					counter = 0
-				}
-				return counter
-			}).Should(Equal(10))
-
-			Consistently(func() string {
-				return helpers.CurlAppRoot(Config, appName)
-			}).ShouldNot(ContainSubstring("Hi, I'm Dora"))
+			for range numberOfAppCurlChecks {
+				Expect(helpers.CurlAppRoot(Config, appName)).To(ContainSubstring("Hello from a staticfile"))
+			}
```

Comment:
If the intent is to check that it succeeds some number of times rather than over some time period, can we parallelize these requests to potentially speed up the tests?

Reply:
Good idea, will make them run in parallel

Reply:
Added